gradle-plugin/README.md

```diff
@@ -0,0 +1,137 @@
+# ProtoData Gradle plugin
+
+This Gradle plugin allows Java developers launch ProtoData without extra CLI commands.
+
+## Usage
+
+### Applying the plugin
+
+To apply the plugin to the project, use the `plugins { }` block syntax.
+
+```kotlin
+plugins {
+    id("io.spine.proto-data") version("<ProtoData version>")
+}
+```
+
+Or, alternatively, use the old-fashioned `buildscript { }` syntax.
+
+```kotlin
+buildscript {
+    repositories {
+        gradlePluginPortal()
+        mavenCentral()
+    }
+    
+    dependencies {
+        classpath("io.spine:proto-data:<ProtoData version>")
+    }
+}
+
+apply(plugin = "io.spine.proto-data")
+```
+
+See the plugin [homepage](https://plugins.gradle.org/plugin/io.spine.proto-data) for more.
+
+### Installing and launching ProtoData
+
+When using ProtoData for the first time, you will have to install the executable.
+To do so, run:
+```
+./gradlew installProtoData
+```
+After running the task, the ProtoData executable might not turn up in the `PATH` environment
+variable at once. The command will print instructions to finalize the installation.
+
+If your environment goes not allow you to execute those instructions, e.g., on a CI instance, you
+can specify the ProtoData installation location by hand:
+```
+./gradlew installProtoData -PprotoDataLocation="my/custom/path"
+```
+
+Now, as ProtoData is installed, simply run the build as usual. ProtoData will be launched
+automatically before Java compilation.
+
+If you used the custom installation path, just add the same Gradle property to all the other Gradle
+invocations. For example:
+```
+./gradlew build -PprotoDataLocation="my/custom/path"
+```
+
+See [this doc](https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_configuration_properties)
+for more ways to define a Gradle property to avoid extra CLI arguments.
+
+### Configuration
+
+ProtoData needs Renderers and Plugins to be useful. You can specify those and more via a Gradle
```

Comment:
Something like "ProtoData requires Renderers and Plugins for meaningful operation." would be closer to what you meant.